### PR TITLE
UI fixes and Menu Burger Fixes

### PR DESCRIPTION
**Pull request by nekomeowww**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,2 +1,3 @@
-REACT_APP_FE_REPO=https://github.com/NaoMikuOshi/Matataki-Token-AirDropper
```
Comment (NaoMikuOshi) [inline .env:1]: Please do not delete this

File: .env [hunk 2]
```diff
@@ -1,2 +1,3 @@
...
+REACT_APP_MATATAKI_API=https://apitest.smartsignature.io
```
Comment (NaoMikuOshi) [inline .env:1]: Already defined in `.env.development`

File: .env [hunk 3]
```diff
@@ -1,2 +1,3 @@
...
+REACT_APP_BACKEND_API=https://apitest.matataki-airdrop.xyz
```
Comment (NaoMikuOshi) [inline .env:2]: Already defined in  `.env.development`

**Comment by nekomeowww** [comment]
Added

File: src/components/Navigation.jsx [hunk 4]
```diff
@@ -36,8 +38,20 @@ export default function Navigation() {
   const store = useStore();
   const isLogined = Boolean(store.get("accessToken"));
 
+  const [active, setActive] = useState(false);
+
+  let handleClick = () => {
+    console.log(active);
+    setActive({ active: !active });
```
Comment (NaoMikuOshi) [inline src/components/Navigation.jsx:45]: My bad. It should be
```js
setActive(!active);
```

**Pull request merged**